xla/service/cpu/ir_emitter.cc

```diff
@@ -109,14 +109,24 @@ using llvm_ir::SetToFirstInsertPoint;
 
 namespace cpu {
 
+bool IsNativeConvertSupportedOnTargetCPU(std::string feature_string) {
+  if (absl::StrContains(feature_string, "+avxneconvert") ||
+      absl::StrContains(feature_string, "+amx-bf16")) {
+    return true;
+  }
+  return false;
```

Comment:
One more nit please, this can be simplified to:
```suggestion
  return absl::StrContains(feature_string, "+avxneconvert") ||
         absl::StrContains(feature_string, "+amx-bf16");
```

Reply:
Done